web/src/elements/ak-checkbox-group/ak-checkbox-group.ts

```diff
@@ -112,10 +113,14 @@ export class CheckboxGroup extends AkElementWithCustomEvents {
     @queryAll('input[type="checkbox"]')
     checkboxes!: NodeListOf<HTMLInputElement>;
 
+    @state()
```

Comment:
Storing the internal "values" independently of the "value" field, which is really just `defaultValue` for our purposes.